Packs/Gmail/Integrations/Gmail/Gmail.yml

```diff
@@ -61,6 +73,16 @@ script:
     description: Deletes a Gmail user.
     execution: false
     name: gmail-delete-user
+    outputs:
+    - contextPath: GmailLabel.UserID
+      description: The UserID the label belongs to.
+      type: String
+    - contextPath: GmailLabel.Name
+      description: The name of the label.
+      type: String
+    - contextPath: GmailLabel.ID
+      description: The label ID.
+      type: String
```

Comment:
I think the definition of the context output here is under the ```gmail-delete-user``` which is wrong. You should add all of these under line 63.

Reply:
Darn it, yes, I copied into the wrong section. I hate YML.